not_used_files/usda_native.R

```diff
@@ -0,0 +1,33 @@
+sp = "Abutilon incanum"
```

Comment:
Done!

(I think, I was not sure about the use of GET) I removed library call's
(but do I need to add those somewhere else?)

Should I need to PR again?

Thanks for the comments
Nacho

On Fri, May 8, 2015 at 6:06 PM, Scott Chamberlain <notifications@github.com>
wrote:

> In not_used_files/usda_native.R
> <https://github.com/ropensci/traits/pull/28#discussion_r29952402>:
>
> > @@ -0,0 +1,33 @@
> > +sp = "Abutilon incanum"
>
> @ibartomeus <https://github.com/ibartomeus> can you delete this file, or
> do you need it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ropensci/traits/pull/28/files#r29952402>.
>



-- 
Ignasi Bartomeus PhD
www.bartomeuslab.com
@ibartomeus
Skype: nachobartomeus

Dpto. Ecología Integrativa
Estación Biológica de Doñana (EBD-CSIC)
Avda. Américo Vespucio s/n
Isla de la Cartuja
41092, Sevilla (Spain)